frontend/src/editor/cell/cell-actions.tsx

```diff
@@ -0,0 +1,255 @@
+            <PlusCircleIcon size={13} strokeWidth={1.5} />
+            <ChevronUpIcon size={12} strokeWidth={1.5} />
+          </MultiIcon>
+        ),
+        label: "Create cell above",
+        hotkey: "cell.createAbove",
+        handle: () => createNewCell(cellId, /*before=*/ true),
+      },
+      {
+        icon: (
+          <MultiIcon>
+            <PlusCircleIcon size={13} strokeWidth={1.5} />
+            <ChevronDownIcon size={12} strokeWidth={1.5} />
+          </MultiIcon>
+        ),
+        label: "Create cell below",
+        hotkey: "cell.createBelow",
+        handle: () => createNewCell(cellId, /*before=*/ false),
+      },
+      {
+        icon: <ChevronUpIcon size={13} strokeWidth={1.5} />,
+        label: "Move cell up",
+        hotkey: "cell.moveUp",
+        handle: () => moveCell(cellId, /*before=*/ true),
+      },
+      {
+        icon: <ChevronDownIcon size={13} strokeWidth={1.5} />,
+        label: "Move cell down",
+        hotkey: "cell.moveDown",
+        handle: () => moveCell(cellId, /*before=*/ false),
+      },
+      {
+        icon: (
+          <MultiIcon>
+            <FocusIcon size={13} strokeWidth={1.5} />
+            <ChevronUpIcon size={12} strokeWidth={1.5} />
+          </MultiIcon>
+        ),
+        label: "Focus cell above",
+        hotkey: "cell.focusUp",
+        handle: () => focusCell(cellId, /*before=*/ true),
+      },
+      {
+        icon: (
+          <MultiIcon>
+            <FocusIcon size={13} strokeWidth={1.5} />
+            <ChevronDownIcon size={12} strokeWidth={1.5} />
+          </MultiIcon>
+        ),
+        label: "Focus cell below",
+        hotkey: "cell.focusDown",
+        handle: () => focusCell(cellId, /*before=*/ false),
+      },
+      {
+        icon: <ChevronsUpIcon size={13} strokeWidth={1.5} />,
+        label: "Send to top",
+        hotkey: "cell.sendToTop",
+        handle: () => sendToTop(cellId),
+      },
+      {
+        icon: <ChevronsDownIcon size={13} strokeWidth={1.5} />,
+        label: "Send to bottom",
+        hotkey: "cell.sendToBottom",
+        handle: () => sendToBottom(cellId),
+      },
+    ],
+
+    // Delete
+    [
+      {
+        label: "Delete",
+        variant: "danger",
+        icon: <Trash2Icon size={13} strokeWidth={1.5} />,
+        handle: () => deleteCell(cellId),
+      },
+    ],
+  ];
+
+  return (
+    <Popover open={open} onOpenChange={setOpen}>
+      <TooltipRoot delayDuration={400} disableHoverableContent={true}>
+        {!open && (
+          <TooltipContent
+            className="dark dark-theme w-full bg-card"
+            tabIndex={-1}
+          >
+            <div className="text-xs text-foreground-muted flex flex-col text-center">
+              <span>
+                <span className="text-foreground font-semibold">Drag </span>to
+                move cell
+              </span>
+              <span>
+                <span className="text-foreground font-semibold">Click </span>to
```

Comment:
Can we allow the user to open the menu with `Enter` too, if they have tabbed to focus on it?

Not urgent, can do in a follow-up, up to you

Reply:
ill do it in a followup